stablehlo/dialect/TypeInference.cpp

```diff
@@ -3272,6 +3323,11 @@ LogicalResult verifyBroadcastInDimOp(std::optional<Location> location,
     return success();
   }
 
+  // broadcast_in_dim_c1
+  if (failed(isSameQuantScaleZeroPoint(location, operand.getType(),
```

Comment:
```
(C1) element_type(result) is given by:
element_type(operand), if !is_per_axis_quantized(operand).
element_type(operand) except that quantization_dimension(operand), scales(operand), and zero_points(operand) may differ from quantization_dimension(result), scales(result), and zero_points(result) resp., otherwise.
```

whya re we macthing scales and zero-points here as broadcast_in_dim does not need that?